proposals/description/fip_56.ts

```diff
@@ -0,0 +1,134 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_56: ProposalDescription = {
+  title: 'FIP-56: Fei v2 Deployment',
+  commands: [
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0x591560f4b82b12ea68e074d47d2fecc152ba0ba0bb5d01b9d622a13a84c2bb5d',
+        '0x899bd46557473cb80307a9dabc297131ced39608330a2d29b2d52b660c03923e'
+      ],
+      description: 'Create TRIBE_MINTER_ADMIN Role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: ['0x591560f4b82b12ea68e074d47d2fecc152ba0ba0bb5d01b9d622a13a84c2bb5d', '{tribeReserveStabilizer}'],
+      description: 'Grant TribeReserveStabilizer Admin Role'
+    },
+    {
+      target: 'tribeMinter',
+      values: '0',
+      method: 'setContractAdminRole(bytes32)',
+      arguments: ['0x591560f4b82b12ea68e074d47d2fecc152ba0ba0bb5d01b9d622a13a84c2bb5d'],
+      description: 'Set TribeMinter Contract Admin Role'
+    },
+    {
+      target: 'tribe',
+      values: '0',
+      method: 'setMinter(address)',
+      arguments: ['{tribeMinter}'],
+      description: 'Grant TRIBE_MINTER the Tribe Minter role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantMinter(address)',
+      arguments: ['{daiPSM}'],
+      description: 'Grant Minter Role to DAI PSM'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantPCVController(address)',
+      arguments: ['{daiPCVDripController}'],
+      description: 'Give the DAI PCVDripController the PCVController role so that it can withdraw from Compound'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0x1749ca1ca3564d20da6efea465c2a5ae869a9e4b006da7035e688beb14d704e0',
+        '0x899bd46557473cb80307a9dabc297131ced39608330a2d29b2d52b660c03923e'
+      ],
+      description: 'Create PSM_ADMIN_ROLE'
+    },
```

Comment:
This is not granted to anyone ?

Reply:
No, it is not granted to anyone but it is created and will be used later.